x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/slm/SlmHealthIndicatorService.java

```diff
@@ -62,11 +65,18 @@ public HealthIndicatorResult calculate(boolean includeDetails) {
                 Collections.emptyList()
             );
         } else if (slmMetadata.getOperationMode() != OperationMode.RUNNING) {
+            List<HealthIndicatorImpact> impacts = Collections.singletonList(
+                new HealthIndicatorImpact(
+                    3,
+                    "Scheduled snapshots are not running, which could lead to future data loss.",
```

Comment:
```suggestion
                    "Scheduled snapshots are not running.  Enable schedule snapshot management to prevent loss of data in the future.",
```
I don't think disabling SLM **leads** to future data loss 😛 , more of insurance to prevent data loss... Not sure how to best word this?